game.py

```diff
@@ -37,251 +37,301 @@
-        self.WIN.blit(head3, (780, 20))
-
-        # BLACK RECTANGLES DISPLAY
-        pygame.draw.line(self.WIN, BLACK, (350, 0), (350, 768), 1)
-        pygame.draw.line(self.WIN, BLACK, (840, 0), (840, 768), 1)
-        pygame.draw.rect(self.WIN, BLACK, (0, 0, 350, 768))
-        pygame.draw.rect(self.WIN, BLACK, (840, 0, 693, 768))
-
     def create_object_list(self):
         if (self.tick_counter == 10 or self.tick_counter == 35) and self.move == 1:
-            self.current = Element()
-            if self.last.x < 530:
+            self.soon_to_draw = Element(game)
+            if self.last.x_to_draw < self.MIDDLE_OF_SCREEN_X:
                 while True:
-                    self.current = Element()
-                    if self.current.x > 530 and self.current.element != self.lastright.element:
+                    self.soon_to_draw = Element(game)
+                    if self.soon_to_draw.x_to_draw > self.MIDDLE_OF_SCREEN_X and self.soon_to_draw.element != self.lastright.element:
                         break
-                self.last = self.lastright = self.current
+                self.last = self.lastright = self.soon_to_draw
                 self.objectlist.append(self.lastright)
             else:
                 while True:
-                    self.current = Element()
-                    if self.current.x < 530 and self.current.element != self.lastleft.element:
+                    self.soon_to_draw = Element(game)
+                    if self.soon_to_draw.x_to_draw < self.MIDDLE_OF_SCREEN_X and self.soon_to_draw.element != self.lastleft.element:
                         break
-                self.last = self.lastleft = self.current
+                self.last = self.lastleft = self.soon_to_draw
                 self.objectlist.append(self.lastleft)
 
     def update_car_angles(self):
-        if self.leftmove == 0:
+        if self.left_moved == 0:
             if self.left_angle > 0:
-                self.left_angle -= self.anglespeed
+                self.left_angle -= ANGLE_SPEED
             if self.left_angle < 0:
-                self.left_angle += self.anglespeed
+                self.left_angle += ANGLE_SPEED
 
-        if self.rightmove == 0:
+        if self.right_moved == 0:
             if self.right_angle > 0:
-                self.right_angle -= self.anglespeed
+                self.right_angle -= ANGLE_SPEED
 
             if self.right_angle < 0:
-                self.right_angle += self.anglespeed
+                self.right_angle += ANGLE_SPEED
 
     def update_car_positions(self):
-        if self.leftmove == 1:
+        LEFT_CAR_LEFT_BOUND = self.lane_1 - self.car_center  # How far the car can go left
+        LEFT_CAR_RIGHT_BOUND = self.lane_2 - self.car_center   # How far the car can go right
+        RIGHT_CAR_LEFT_BOUND = self.lane_3 - self.car_center   # How far the car can go left
+        RIGHT_CAR_RIGHT_BOUND = self.lane_4 - self.car_center   # How far the car can go right
```

Comment:
If these are to be used only here then name them accordingly.

Reply:
Will do